.pre-commit-config.yaml

```diff
@@ -104,6 +104,7 @@ repos:
                 aiida/orm/utils/links.py|
                 aiida/plugins/entry_point.py|
                 aiida/plugins/factories.py|
+                aiida/plugins/utils.py|
```

Comment:
I guess this can be removed?

Reply:
Why? This list of files is to explicitly *enable* type-checking. Since I fixed the type hinting, I added the file to this list.

Reply:
Ah, sorry, I thought this is to exclude the file.